### PR TITLE
EPMDJ-3644: add a stub with empty scopes

### PR DESCRIPTION
**Pull request by SMPlay**
Issue: EPMDJ-3644

### CODE REVIEW

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-    text-align: center;
-    color: map-get($color-palette, middle-blue);
-  }
-}
+@import '~common/style-guide/index';
+
+.no-scope-stub {
+  display: grid;
+  place-items: center;
+  height: 80%;
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss:6]: cacl()

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/scopes-list/scopes-list.tsx [hunk 2]
```diff
@@ -56,177 +57,182 @@ export const ScopesList = scopesList(({ className }: Props) => {
...
-            Cell={({
-              value, item: {
-                id, started, active, enabled, finished,
-              },
-            }) => (
-              <NameCell
-                onClick={() => push(`/full-page/${agentId}/${buildVersion}/${pluginId}/scopes/${id}`)}
-                data-test="scopes-list:scope-name"
+        {
+          scopesData.length !== 0
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/scope/scopes-list/scopes-list.tsx:61]: scopesData.length > 0

**Review by RomanAchilov — APPROVED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/scopes-list/scopes-list.tsx [hunk 3]
```diff
@@ -56,177 +57,182 @@ export const ScopesList = scopesList(({ className }: Props) => {
           All scopes
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/scope/scopes-list/scopes-list.tsx:57]: Scopes

**Review by RomanAchilov — APPROVED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
-  &__title {
-    margin-top: 40px;
-    margin-bottom: 8px;
-    font-size: 20px;
-    line-height: 32px;
-    text-align: center;
-    color: map-get($color-palette, middle-blue);
-  }
-}
+@import '~common/style-guide/index';
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss:1]: line endings

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.tsx [hunk 5]
```diff
@@ -11,11 +12,13 @@ const noScopeStub = BEM(styles);
 
 export const NoScopeStub = noScopeStub(({ className }: Props) => (
   <div className={className}>
-    <Content>
-      <Title>No active scope for this build</Title>
-    </Content>
+    <Panel direction="column">
+      <Icons.Scope width={157} height={157} data-test="no-scope-stub:test-icon" />
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.tsx:16]: the commit message said 'No alignment in the "All scopes" table header when it is empty'. So WTF is this then?

**Review by pavyarov — CHANGES_REQUESTED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.tsx (hunk 5 above)
Comment (SMPlay) [reply src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.tsx:16]: that was the name of the task in JIRA. Should I change to "add a stub with empty scopes"?

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss (hunk 4 above)
Comment (SMPlay) [reply src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss:1]: I can't resolve this

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.tsx (hunk 5 above)
Comment (pavyarov) [reply src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.tsx:16]: of course! commit message and PR description !== jira header

File: src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss (hunk 4 above)
Comment (pavyarov) [reply src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss:1]: em... What the problem? Turn off `autocrlf`

Comment (pavyarov) [reply src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss:1]: Also check husky, husky should format only staged files

Comment (SMPlay) [reply src/pages/agent-full-page/coverage-plugin/scope/no-scope-stub/no-scope-stub.module.scss:1]: My autocrlf is off

**Review by pavyarov — APPROVED** [review]

**Pull request merged**